src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -24,4 +24,14 @@ class LeoRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with
       status shouldEqual StatusCodes.OK
     }
   }
+
+  it should "200 when deleting a cluster" in {
```

Comment:
This tests deleting a cluster that does not exist - do we want 200 in that case?

I would like to see test cases for both clusters that exist and those that don't.

Reply:
404 when deleting something that doesn't exist.